app/components/ContentEditable.tsx

```diff
@@ -81,6 +81,17 @@ const ContentEditable = React.forwardRef(
       }
     }, [value, ref]);
 
+    // Ensure only plain text can be pasted into title when pasting from another
```

Comment:
Moving standard handling and styling down into this component from `EditableTitle` for re-use.